WcaOnRails/app/jobs/timed_application_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class TimedApplicationJob < ApplicationJob
```

Comment:
The Rails 5 upgrade instructions actually told me to create this intermediate, empty class. See http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#activejob-now-inherits-from-applicationjob-by-default. The instructions say "you **need** to create an application_job.rb" (emphasis mine). I did not attempt to break their rules here, so we have an empty application_job.rb =)